Data/Vinyl/ARec/Internal.hs

```diff
@@ -23,9 +24,26 @@
 -- element to the head of a list, but element access is linear time;
 -- array access time is uniform, but extending the array is more
 -- slower.
+
```

Comment:
This blank line needs to start with `-- ` or `haddock` can not process the file.